src/pybind/mgr/dashboard/frontend/e2e/helper.po.ts

```diff
@@ -1,9 +1,16 @@
 import { browser } from 'protractor';
+import { PoolPageHelper } from './pools/pools.po';
 
 export class Helper {
   static EC = browser.ExpectedConditions;
   static TIMEOUT = 10000;
 
+  pools: PoolPageHelper;
+
+  constructor() {
+    this.pools = new PoolPageHelper();
+  }
+
```

Comment:
Do you anticipate other constructors needing to be added to the Helper class, or is this an edge case?

Reply:
It's the actual idea that all helpers are supposed to be instantiated here to make them globally available in all suites. This is not a good OOP pattern, but the idea is to make all code in all helpers reusable across all suites here by default. For instance, The RBD suite is going to depend on a pool, which can be created, deleted and its existance verified by using the PoolHelper instantiated here.

But now that you've questioned it, it might not be necessary for many/most helpers and hence, might not be such a good idea. But that, I think, depends on the amount of methods required from helpers in other suites, which I can not foresee. I'm open for suggestions, though.

Reply:
Thank you for the explanation - I think for now it makes sense to leave, and we can always look into refactoring in the future once the test suite is more sophisticated and see if another solution presents itself.